firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/log/LogFileManager.java

```diff
@@ -142,6 +146,11 @@ public void writeToLog(long timestamp, String msg) {}
       return null;
     }
 
+    @Override
+    public String getLogAsString() {
+      return null;
```

Comment:
I'm not sure I understand why this is null

Reply:
Look at the context - this is a no-op implementation.

Reply:
ahh, should it be returning "" then? (Same story with the bytes one -- should it be returning an empty array?) (Should we document the expected outputs in the FileLogStore interface?)

Reply:
Good opportunities for improvement. I'll add a TODO.

Reply:
Also, to answer your question, no - the intent is that `null` means there is no data, separate from a potentially empty log line `""`.